sleap/io/dataset.py

```diff
@@ -2447,21 +2467,20 @@ def numpy(
             tracks = np.full((n_frames, n_tracks, n_nodes, 2), np.nan, dtype="float32")
         for lf in lfs:
             i = lf.frame_idx - first_frame
+            lf_insts: Union[List[Instance], List[PredictedInstance]] = (
```

Comment:
`List[Union[Instance, PredictedInstance]]` for mixed lists?

Reply:
If a labeled frame has ANY user instances in the frame, then we ONLY use the user instances (the frame is already completely labeled). The lists are never mixed.